### ISSUE TITLE
Mixins don't get considered when considering aliveness of a property

### ISSUE THREAD

**Issue by minichate** [minichate/ember-cli-tree-shake#3]

**Comment by anurag90x** [comment]
The same is true for controller/component inheritance. Would it make sense to make the computed properties of each be a union of the controller/component's properties + all the inherited properties (mixins + regular ones)?

**Comment by minichate** [comment]
Yes it would! The tricky part is actually following the inheritance structure and figuring out what `.js` file to parse into a CST I think. I haven't taken a good hard look at it yet. PRs welcome :)
